doc/faq.rst

```diff
@@ -18,3 +18,32 @@ The sync client has been intentionally limited to sync no deeper than
 fifty sub-directories, to help prevent memory problems. 
 Unfortunately, it, *currently*, does not report an error when this occurs. 
 However, a UI notification is planned for a future release of ownCloud.
+
+I want to move my local sync folder
+-----------------------------------
+
+The ownCloud desktop client does not provide a way to change the local sync directory. 
+However, it can be done, though it is a bit unorthodox. 
+Specifically, you have to:
+
+1. Remove the existing connection which syncs to the wrong directory
+2. Add a new connection which syncs to the desired directory
+
+image:: images/setup/ownCloud-remove_existing_connection.png
+
+To do so, in the client UI, which you can see above, click the "**Account**" drop-down menu and then click "Remove". 
+This will display a "**Confirm Account Removal**" dialog window.
+
+image:: images/setup/ownCloud-remove_existing_connection_confirmation_dialog.png
+
+If you're sure, click "**Remove connection**".
+
```

Comment:
At this point you can actually move/copy the old local sync folder (which has all your files locally) to the new place that you want it. Then when you "Add new" sync connection, choose "keep existing data". 
ownCloud client happily checks the files in the newly-added sync folder and finds that they match with what is on the server.
The alternative is to start with an empty new sync folder - which means that the ownCloud client will have to download your files again. That may be many GB that is already in the old sync folder on your system.
I haven't tried, but I guess copying over from the old local folder will also preserve the selection of folders to be ignored... since the "hidden" oC database will come across from the old local folder.